testpilot/frontend/static-src/app/views/experiment-page.js

```diff
@@ -273,7 +267,10 @@ export default PageView.extend({
       eventLabel: 'Disable Experiment'
     });
 
-    const queryParams = querystring.stringify({ref: 'disable', experiment: this.model.title});
+    const installed = Object.keys(app.me.installed);
```

Comment:
Oh, I see what's up... `app.me.installed` gets set first by [the server-side API result (as an Array)](https://github.com/mozilla/testpilot/blob/master/testpilot/users/views.py#L40) and later by [a message from the add-on (as an Object)](https://github.com/mozilla/testpilot/blob/master/testpilot/frontend/static-src/app/models/me.js#L45). I think we need to change one of these sources to be consistent with the other